### PR TITLE
Have BugsnagMiddleware for Django inherit from Django MiddlewareMixin…

### PR DESCRIPTION
**Pull request by jonnypickett**
… for 1.10 compatibility

### CODE REVIEW

File: bugsnag/django/middleware.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
 from __future__ import division, print_function, absolute_import
 
+from django.utils.deprecation import MiddlewareMixin
```
Comment (kattrali) [inline bugsnag/django/middleware.py]: To make this backwards compatible, this call should be wrapped to ensure no import error occurs and there is a fallback base class. Something like:

```python
try:
    from django.utils.deprecation import MiddlewareMixin
except ImportError:
    MiddlewareMixin = object
```

Comment (jonnypickett) [inline bugsnag/django/middleware.py]: Good catch. But actually now that you point this out, passing a parameter to `object.__init__()` throws `TypeError: object.__init__() takes no parameters`, which is what will happen for Django projects < 1.10 with passing `get_response` to the parent `__init__` method.

Comment (jonnypickett) [inline bugsnag/django/middleware.py]: Great. Glad I could help.

File: bugsnag/django/utlis.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from __future__ import print_function, absolute_import
```
Comment (kattrali) [inline bugsnag/django/utlis.py]: Oh, just noticed the typo in the filename

**Pull request merged**